tests/test-quickstart/src/test/java/org/eclipse/jetty/quickstart/AttributeNormalizerTest.java

```diff
@@ -82,6 +92,16 @@
         env.put("WAR", asTargetPath(title,"app%2Fnasty/base/webapps/FOO"));
         
         data.add(new Object[]{arch, title, env});
+
+        // ------
+        title = "Root Path Setup";
+        env = new HashMap<>();
+        env.put("jetty.home", "/");
+        env.put("jetty.base", "/");
+        env.put("WAR", asTargetPath(title,"webapps/root"));
+        
+        data.add(new Object[]{arch, title, env});
+        
```

Comment:
This should be OS neutral.  Consider ..

``` java
        title = "Root Path Setup";
        env = new HashMap<>();
        FileSystem fs = MavenTestingUtils.getTargetPath().getFileSystem();
        Path rootPath = fs.getRootDirectories().iterator().next();
        env.put("jetty.home", rootPath.toString());
        env.put("jetty.base", rootPath.toString());
        env.put("WAR", rootPath.resolve("webapps/root").toString());
```

Reply:
Slightly simpler version (and more correct on Windows if running on say `D:\code`)

``` java
        title = "Root Path Setup";
        env = new HashMap<>();
        Path rootPath = MavenTestingUtils.getTargetPath().getRoot();
        env.put("jetty.home", rootPath.toString());
        env.put("jetty.base", rootPath.toString());
        env.put("WAR", rootPath.resolve("webapps/root").toString());
```